### DISCUSSION TITLE
HackRF portapack

### DISCUSSION THREAD

**Discussion by 7omoud** [Oros42/IMSI-catcher#147] [General]
I just bought HackRF, and I like it and like all the comments and helps I saw in this community. Many things to do with this great device! But I have two questions, first can the device read NFC card? And my second question, can the device get my cellphone frequency? 
By the way im new in this world 🙈

**Comment by Oros42** [comment]
For NFC, I don't know. Maybe yes.
«can the device get my cellphone frequency?» : Yes

**Reply by 7omoud** [reply]
How? Can you show me the way?

**Reply by 7omoud** [reply]
Ops forgot to mention! My device is HackRF portapack H2 :)
